### ISSUE TITLE
update

### ISSUE THREAD

**Issue by helpmebro911** [adamhrv/skylift#20]
hello
you plan for update?

**Comment by adamhrv** [comment]
If the project is useful to others then I'm happy to update and maintain. Any feedback on how it's being used and whether it still works would be motivating.

**Comment by helpmebro911** [comment]
https://github.com/adamhrv/skylift#todo    
Well, I and many other people would find the features you planned to add useful.
I'm especially looking forward to it.

` add BLE adverts

 add randomness to WiFi packets

 add Wigle API`

**Comment by helpmebro911** [comment]
https://t.me/virtual_router/1
there is also such software that allows you to automate the firmware of the board for spoofing.
perhaps you can also make a convenient GUI interface for your software

**Comment by helpmebro911** [comment]
perhaps the interface can be based on https://github.com/davesc63/GeoPort?

**Comment by helpmebro911** [comment]
add 5.8 board support
https://aliexpress.com/item/4001315055682.html?sku_id=10000015697348615

**Comment by helpmebro911** [comment]
or maybe it is possible to make software as a module for https://github.com/arendst/Tasmota
then maybe it will be possible to implement a graphical configurator without using a personal computer

**Comment by adamhrv** [comment]
BLE broadcasts would be the next priority. But can you tell me if you're able to successfully geospoof your location using wifi? It's been 8 years since the project code was posted publicly and I had assumed wifi geolocation service overlords would have tried to patch this by now.

**Comment by helpmebro911** [comment]
Yes, I wrote a link above to software that is frequently updated and has this function.
but it has a closed source code ((
https://t.me/virtual_router/1

**Comment by helpmebro911** [comment]
> BLE broadcasts would be the next priority. But can you tell me if you're able to successfully geospoof your location using wifi? It's been 8 years since the project code was posted publicly and I had assumed wifi geolocation service overlords would have tried to patch this by now.

news?

**Comment by helpmebro911** [comment]
news?
